### PR TITLE
Change visibility of method assertInstanceOfApp

### PR DESCRIPTION
**Pull request by abbadon1334**
allow override of method.

### CODE REVIEW

**Comment by PhilippGrashoff** [comment]
Like it, had some trouble with other private methods lately.

File: src/AppScopeTrait.php [hunk 1]
```diff
@@ -63,7 +63,7 @@ trait AppScopeTrait
      */
     public $unique_hashes = [];
 
-    private function assertInstanceOfApp(object $app): void
+    protected function assertInstanceOfApp(object $app): void
```
Comment (mvorisek) [inline src/AppScopeTrait.php:66]: https://3v4l.org/2qPLn :D

please try itself before

**Review by mvorisek — CHANGES_REQUESTED** [review]

File: src/AppScopeTrait.php (hunk 1 above)
Comment (abbadon1334) [reply src/AppScopeTrait.php:66]: i need to override it in class not use it as it is, did you know what are you doing?

If i have to rewrite even traits, user can use another framework, no? I like private but not in traits, too much trouble. machine cycle have a cost.

**Review by mvorisek — APPROVED** [review]
well, makes sense that it is not possible to overwrite it easily for framework classes otherwise

**Pull request merged**